### PR TITLE
added 'other operations' section and certs page

### PR DESCRIPTION
**Pull request by andrewh1978**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: "Certificates as Kubernetes Secrets"
+hidden: false
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md]: By default, it's not hidden. So this line is not required.

**Comment by harsh-px** [comment]
@andreipope please review

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+title: "Certificates as Kubernetes Secrets"
+keywords: certificates, certs, portworx, kubernetes
+description: Certificates as Kubernetes secrets.
+---
+
+Sometimes it is necessary to store an SSL certificate as a Kubernetes secret. The example here is to secure a third-party S3-compatible objectstore for use with Portworx.
+
+### Creating the secret
+
+ * Copy your certificate to the Kubernetes master node. We will call it `objectstore.pem` and copy it to a folder `/opt/certs`.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md]: Don't need to copy it to the master node. Just any system that where you can run `kubectl` for that cluster.

Comment (andrewh1978) [reply content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md]: Fair point, changed.

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+title: "Certificates as Kubernetes Secrets"
+keywords: certificates, certs, portworx, kubernetes
+description: Certificates as Kubernetes secrets.
+---
+
+Sometimes it is necessary to store an SSL certificate as a Kubernetes secret. The example here is to secure a third-party S3-compatible objectstore for use with Portworx.
+
+### Creating the secret
+
+ * Copy your certificate to somewhere `kubectl` is configured for this Kubernetes cluster. We will call it `objectstore.pem` and copy it to a folder `/opt/certs`.
```
Comment (andreipope) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md]:
```suggestion
 * Copy your certificate to somewhere `kubectl` is configured for this Kubernetes cluster. We will call the file `objectstore.pem` and copy it to the `/opt/certs` folder.
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ * Create the secret:
+
+```bash
+$ kubectl -n kube-system create secret generic objectstore-cert --from-file=/opt/certs/
```
Comment (andreipope) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md]: Let's make it pastable:

```suggestion
kubectl -n kube-system create secret generic objectstore-cert --from-file=/opt/certs/
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+ * Confirm it was created correctly:
+
+```bash
+$ kubectl -n kube-system describe secret objectstore-cert
```
Comment (andreipope) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md]:
```suggestion
kubectl -n kube-system describe secret objectstore-cert
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+ * Update the Portworx daemonset to add the mount secret and the environment variable to read the certificate:
```
Comment (andreipope) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md]: It's a bit confusing.

What do you think about this?
```suggestion
 * Update the Portworx daemonset to add the mount secret and the environment variable:
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+```bash
+$ kubectl -n kube-system edit ds portworx
```
Comment (andreipope) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/other-operations/certs.md]: Let's make it pastable:

```suggestion
kubectl -n kube-system edit ds portworx
```

**Comment by andreipope** [comment]
> @andreipope please review

@harsh-px Done. Thank you!

**Comment by harsh-px** [comment]
Still not convinced if we need to add a generic new operations section. I'll need to think if there is a better organization here.

**Pull request merged**